src/pages/STG_PanelAddrVerification.page

```diff
@@ -2,19 +2,19 @@
 
```

Comment:
when you go into Edit mode on the Address page, the Edit button remains visible in the header.  The save and cancel buttons do correctly display at the bottom.